db/migrate/20220520210546_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :name
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, null: false, foreign_key: true
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ]  I  can see you added name as a string here instead of title. Make sure you update it as per the previous Code Reviewer.